app/modules/pullRequests/saga/task/submit.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import { type Saga } from 'redux-saga';
+import { call } from 'redux-saga/effects';
+
+import asyncRequests from 'modules/asyncRequests';
+
+import actions from '../../actions';
+import * as a from '../../actionTypes';
+
+const { putAndReturn } = asyncRequests.lib;
+
+const submit = function* (action: a.SubmitAction): Saga<{ id: string }> {
+  const { message, sourceTopicId, targetTopicId, userId } = action.payload;
+
+  // Create the new pull request in the backend.
+  const { id } = yield call(putAndReturn, actions.apiPost(
+    message, sourceTopicId, targetTopicId, userId,
+  ));
+
```

Comment:
Shouln't the saga also fetch the newly created pull request from the backend so that it's included in the redux store?

Reply:
In theory, after creation the user would get redirected to the pull request, where the `FetchWrapper` would take care of this. Right now this is not relevant, but in the future the `FetchWrapper` would re-fetch the pull request including the diff to visualize. That's why I'm not fetching it here.

However, I'm in favour of having consistency in these sagas, so perhaps we should always fetch the newly created resource. And write this guideline down somewhere.